### DISCUSSION TITLE
Welcome to Windows-On-Reins Discussions!

### DISCUSSION THREAD

**Discussion by gordonbay** [gordonbay/Windows-On-Reins#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by teeotsa** [comment]
Hey, not sure what to write here so... I'm intrested in tech and uhm... I'm 15 years old :laughing: 

I have some similar projects as you have here ( [Link](https://github.com/teeotsa/windows-11-debloat) if you're intrested! ) 

Not really sure, would be nice to meet some new people!

**Reply by gordonbay** [reply]
your Disable-WindowsDefender.ps1 probably not disabling win def entirely. the only way i found to disable it is safe mode -> delete folders

**Reply by teeotsa** [reply]
> your Disable-WindowsDefender.ps1 probably not disabling win def entirely. the only way i found to disable it is safe mode -> delete folders

hmm, okay. well, my method works well. haha

**Comment by ghost** [comment]
Hi can you help me, I can't logon on my office 365 always says need internet.. Any fix on this?
